doc/source/reference/commands/io_flush.rst

```diff
@@ -310,6 +315,61 @@ and all columns:
 .. include:: ../../example/reference/commands/io_flush/only_opened_yes.log
 .. io_flush --only_opened yes
 
+.. _io-flush-dependent:
+
+``dependent``
+"""""""""""""
+
+.. versionadded:: 9.0.2
+
+Specifies whether not only child objects of the flush target object,
+but also related index columns and table which contains related index
+columns are flushed.
+
+This option is similar to ``recursive``, but there is a different that
+related index columns and table are also flushed.  It is useful for
+users because this option take care of the objects which objects are related.
+It supports not to forget flushing related objects.
+
+``dependent`` value must be ``yes`` or ``no``. ``yes`` means that all
+of the specified flush target object and child objects, related objects.
+``no`` means that only the specified flush target object is flushed.
+
+For example, ``--dependent yes`` is enabled for ``TABLE_NAME``, this
+option executes equivalent to the following commands internally.
+
+- Table and its columns::
+
+    io_flush --target_name TABLE_NAME
+
+- A referenced table::
+
+    io_flush --target_name REFERENCED_TABLE_NAME --recursive no
+
+- A related table of an index column (There is source column in TABLE_NAME)::
+
+    io_flush --target_name TABLE_NAME_OF_INDEX_COLUMN --recursive no
+
+- An related index column (There is source column in TABLE_NAME)::
+
+    io_flush --target_name TABLE_NAME_OF_INDEX_COLUMN.INDEX_COLUMN
+
+The following ``io_flush`` flushes all changes in database, all tables
+and all columns:
+
+.. groonga-command
+.. include:: ../../example/reference/commands/io_flush/dependent_yes.log
```

Comment:
I can't find this file.
Do you have not made this file yet?

Reply:
Sorry, I've fogot it.